diff --git a/node_modules/react-images/dist/react-images.es.js b/node_modules/react-images/dist/react-images.es.js
index bf45cfd..f378c19 100644
--- a/node_modules/react-images/dist/react-images.es.js
+++ b/node_modules/react-images/dist/react-images.es.js
@@ -795,7 +795,8 @@ var View = function View(props) {

   var innerProps = {
     alt: formatters.getAltText({ data: data, index: index }),
-    src: getSource({ data: data, isFullscreen: isFullscreen })
+    src: data.src == null ? getSource({ data: data, isFullscreen: isFullscreen }) : data.src,
+    srcSet: data.srcSet
   };

   return glam(
@@ -808,8 +809,8 @@ var View = function View(props) {
       className: className('view-image', { isFullscreen: isFullscreen, isModal: isModal }),
       css: {
         height: 'auto',
-        maxHeight: '100%',
-        maxWidth: '100%',
+        maxHeight: '100vh',
+        maxWidth: '100vw',
         userSelect: 'none',
         margin: 'auto'
       }
